src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -68,6 +70,7 @@ public HTTPJwtAuthenticator(final Settings settings, final Path configPath) {
         super();
 
         String signingKey = settings.get("signing_key");
```

Comment:
Instead of parsing these out, can this simply be changed from `settings.get("signing_key")` to `settings.getAsList("signing_key")`

If a single value is provided, that method should give back a list of one. Alternatively, you can supply a yml list of values and parsing can be avoided.

Reply:
Sure, I can swap that over if you prefer.